_templates/layout.html

```diff
@@ -8,4 +8,142 @@
+	   new_url="https://www.flipcause.com/widget/"+cause_id
+	   window.open(new_url);
+   }
+   else {
+	   document.getElementById("fc-fade").style.display="block";
+	   document.getElementById("fc-fade").style.webkitAnimation="backfadesin 1s";
+	   document.getElementById("fc-fade").style.animation="backfadesin 1s";
+	   document.getElementById("fc-fade").style.mozAnimation="backfadesin 1s";
+	   document.getElementById("fc-light").style.display="block";
+	   document.getElementById("fc-light").style.webkitAnimation="fadesin 1.5s";
+	   document.getElementById("fc-light").style.animation="fadesin 1.5s";
+	   document.getElementById("fc-light").style.mozAnimation="fadesin 1.5s";
+	   document.getElementById("fc-main").style.display="block";
+	   document.getElementById("fc-main").style.webkitAnimation="fadesin 1.5s";
+	   document.getElementById("fc-main").style.animation="fadesin 1.5s";
+	   document.getElementById("fc-main").style.mozAnimation="fadesin 1.5s";
+	   document.getElementById("fc-close").style.display="block";
+	   document.getElementById("fc-close").style.webkitAnimation="fadesin 1.5s";
+	   document.getElementById("fc-close").style.animation="fadesin 1.5s";
+	   document.getElementById("fc-close").style.mozAnimation="fadesin 1.5s";
+	   document.getElementById("fc-myFrame").style.display="block";
+	   document.getElementById("fc-myFrame").style.webkitAnimation="fadesin 1.5s";
+	   document.getElementById("fc-myFrame").style.animation="fadesin 1.5s";
+	   document.getElementById("fc-myFrame").style.mozAnimation="fadesin 1.5s";
+	   document.getElementById("fc-myFrame").src="https://www.flipcause.com/widget/"+cause_id;
+   }
+   }
+   function close_window() {
+	   document.getElementById("fc-fade").style.display="none";
+	   document.getElementById("fc-light").style.display="none";
+	   document.getElementById("fc-main").style.display="none";
+	   document.getElementById("fc-close").style.display="none";
+	   document.getElementById("fc-myFrame").style.display="none";
+   }
+</script>
+<div id="fc-fade" class="fc-black_overlay" onclick="close_window()"></div>
+<div id="fc-light" class="fc-white_content">
+   <div id="fc-main" class="fc-main-box">
+      <div id="fc-close" class="fc-widget_close" onclick="close_window()"> </div>
+      <iframe id="fc-myFrame" iframe height="580" width="925" style="border: 0; 
+         border-radius:5px 5px 5px 5px; box-shadow:0 0 8px rgba(0, 0, 0, 0.5);" scrolling="no" src="">
+      </iframe>
+   </div>
+</div>
+<!--END Flipcause Main Integration Code-->
 {% endblock %}
+
+{% block footer %}
+<div class="container">
+   <a href="http://www.numfocus.org/" style="float:left; margin-top:10px;">
+      <img src="_static/numfocus-logo-small.png">
+   </a>
+   <div class="row-fluid" style="margin-top:25px;">
+      <ul class="inline pull-left">
+         <li> © Copyright 2017 NumPy developers. </li>
+         <li> Created using <a href="http://sphinx.pocoo.org/">Sphinx</a> 1.5.3. </li>
```

Comment:
You can call `{{ super() }}` (or something like that) to get the automatic footer html.